interop-reactiveStreams/shared/src/main/scala/scalaz/zio/interop/reactiveStreams/Adapters.scala

```diff
@@ -0,0 +1,88 @@
+package scalaz.zio.interop.reactiveStreams
+
+import org.reactivestreams.{ Publisher, Subscriber }
+import scalaz.zio._
+import scalaz.zio.interop.reactiveStreams.SubscriberHelpers._
+import scalaz.zio.stream.{ Stream, Take, ZSink, ZStream }
+
+object Adapters {
```

Comment:
We may want to move the  Scaladoc on the methods here to the methods in the implicit classes on the package object.